src/Honeybee_Generator_Wind_Horizontialaxis.py

```diff
@@ -134,206 +134,212 @@ def checktheinputs(_name,_simpleOrAnalytical,_powerCoefficients,_powerControl,_r
         return -1
         
-    if _ratedWindSpeed == None:
-        print "The field _ratedWindSpeed must be specified!"
+    if ratedWindSpeed_ == None:
+        print "The field ratedWindSpeed_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _ratedWindSpeed must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field ratedWindSpeed_ must be specified!")
         return -1
         
-    if _cutInWindSpeed == None:
-        print "The field _cutInWindSpeed must be specified!"
+    if cutInWindSpeed_ == None:
+        print "The field cutInWindSpeed_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _cutInWindSpeed must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field cutInWindSpeed_ must be specified!")
         return -1
         
-    if _cutOutWindspeed == None:
-        print "The field _cutOutWindspeed must be specified!"
+    if cutOutWindspeed_ == None:
+        print "The field cutOutWindspeed_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _cutOutWindspeed must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field cutOutWindspeed_ must be specified!")
         return -1
         
-    if _maxTipSpeedRatio == None:
-        print "The field _maxTipSpeedRatio must be specified!"
+    if maxTipSpeedRatio_ == None:
+        print "The field maxTipSpeedRatio_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _maxTipSpeedRatio must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field maxTipSpeedRatio_ must be specified!")
         return -1
         
-    if _maxPowerCoefficient == None:
-        print "The field _maxPowerCoefficient must be specified!"
+    if maxPowerCoefficient_ == None:
+        print "The field maxPowerCoefficient_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _maxPowerCoefficient must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field maxPowerCoefficient_ must be specified!")
         return -1
         
-    if _localAvWindspeed == None:
-        print "The field _localAvWindspeed must be specified!"
+    if localAvWindspeed_ == None:
+        print "The field localAvWindspeed_ must be specified!"
         w = gh.GH_RuntimeMessageLevel.Warning
-        ghenv.Component.AddRuntimeMessage(w, "The field _localAvWindspeed must be specified!")
+        ghenv.Component.AddRuntimeMessage(w, "The field localAvWindspeed_ must be specified!")
         return -1
 
-    if _turbinecost == None:
-        print "The field _turbinecost must be specified!"
+    if turbinecost_ == None:
```

Comment:
I'm not sure why you change all this input names. Also you can use a honeybee method to check the inputs for you if you name them correctly. Here is an example: https://github.com/mostaphaRoudsari/Honeybee/blob/master/src/Honeybee_Radiance%20Glass%20Material.py#L90-L91